intera_interface/src/intera_motion_interface/interaction_options.py

```diff
@@ -239,6 +245,29 @@ def set_interaction_control_mode(self, interaction_mode = None):
                 rospy.logerr('The number of interaction_control_mode elements must be 1 or %d',
                              self.n_dim_cart)
 
+    def set_disable_damping_in_force_control(self, disable_damping_in_force_control = None):
```

Comment:
I think you'll get the same effect with a lot less work/code if you just make the default function value `InteractionOptions.default_disable_damping_in_force_control` ( or self.default...., I'm not sure which one it is).

Won't that be a lot cleaner?